### PR TITLE
Visitor can see products sorted in different categories

### PR DESCRIPTION
**Pull request by Alex-Bellamy**
### User Story
```
As a visitor,
In order to choose product more easily
I would like to see the products sorted in categories
```
## Changes proposed in this pull request:
* adding feature test 
* adding implementation code to menu.jsx accordingly
## What I have learned working on this feature:
* pushing data into empty arrays

### CODE REVIEW

File: cypress/fixtures/products.json [hunk 1]
```diff
@@ -2,23 +2,32 @@
...
-          "category": "Starter",
+          "category": "Starters",
           "name": "Pizza",
           "description": "This is a cheesy pizza",
           "price": 70
       },
       {
           "id": 2,
-          
+          "category": "Main Courses",
```
Comment (oliverochman) [inline cypress/fixtures/products.json]: are u sure this is how it will be sent off from the backend?

File: cypress/fixtures/products.json [hunk 2]
```diff
@@ -2,23 +2,32 @@
...
           "id": 3,
+          "category": "Deserts",
           "name": "Noodles",
           "description": "You thought it would come with something else but it doesnt",
           "price": 50
-      }
+      },
+      {
+        "id": 4,
+        "category": "Starters",
```
Comment (oliverochman) [inline cypress/fixtures/products.json]: will the categories be sent off capitalized?

File: cypress/fixtures/products.json (hunk 1 above)
Comment (Cma-B) [reply cypress/fixtures/products.json]: category: [:starters, :maincourses, :desserts, :drinks
that's the category they have made & you were right dessert has two S ;)

File: cypress/fixtures/products.json [hunk 3]
```diff
@@ -1,24 +1,39 @@
...
+      "description": "This is a cheesy pizza",
+      "price": 70,
+      "category": "starters"
+    },
+    {
+      "id": 2,
+      "name": "Falafel",
+      "description": "Veggie food is good for you and the planet",
+      "price": 100,
+      "category": "maincourses"
```
Comment (oliverochman) [inline cypress/fixtures/products.json]: i am pretty sure they changed this to `main_courses`

**Pull request merged**